cocos/physics-2d/box2d/shapes/shape-2d.ts

```diff
@@ -133,24 +137,30 @@ export class b2Shape2D implements IBaseShape {
         return this._fixtures.indexOf(fixture);
     }
 
-    _createShapes (scaleX: number, scaleY: number): b2.Shape[] {
+    //relativePositionX/Y : relative Position from shape to rigid body
+    _createShapes (scaleX: number, scaleY: number, relativePositionX: number, relativePositionY: number): b2.Shape[] {
         return [];
     }
 
     _init () {
         if (this._inited) return;
 
         const comp = this.collider;
+        const scale = comp.node.worldScale;
+        // relative Position from shape to rigid body
+        let relativePosition = new Vec3(0, 0, 0);
```

Comment:
Seems no need to create new Vec3, use Vec3.ZERO instead?

Reply:
yes